xml/soc-operations-maintenance.xml

```diff
@@ -1329,4 +1329,92 @@
    </para>
   </important>
  </section>
+ <section xml:id="sec.bootstrap-galera-cluster-with-missing-node">
+   <!-- https://bugzilla.suse.com/show_bug.cgi?id=1132853 -->
+  <title>Bootstrapping the &mariadb_cluster; with &pacemaker; when a node is missing</title>
+  <para>
+    Pacemaker will not promote a node to master until it received from all
+    nodes the latest <glossterm linked="gloss.galera_sequence_number">Sequence number (seqno)</glossterm>
+    That is a problem when one node of the &mariadb_cluster; is down (eg. due
+    to hardware or network problems) because the Sequence number can not be
```

Comment:
<glossterm linked="gloss.galera_sequence_number">Sequence number (seqno)</glossterm>

Reply:
Or up to you, really.

Reply:
Done